components/dashboard/src/prebuilds/list/PrebuildListItem.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+import { TextMuted } from "@podkit/typography/TextMuted";
+import { Text } from "@podkit/typography/Text";
+import { LinkButton } from "@podkit/buttons/LinkButton";
+import { TableCell, TableRow } from "@podkit/tables/Table";
+import type { Prebuild } from "@gitpod/public-api/lib/gitpod/v1/prebuild_pb";
+import { useConfiguration } from "../../data/configurations/configuration-queries";
+import { prebuildDisplayProps, prebuildStatusIconName } from "../../projects/Prebuilds";
+import dayjs from "dayjs";
+import { cn } from "@podkit/lib/cn";
+import { shortCommitMessage } from "../../projects/render-utils";
+
+type Props = {
+    prebuild: Prebuild;
+};
+export const RepositoryListItem: FC<Props> = ({ prebuild }) => {
+    const created = dayjs(prebuild.status?.startTime?.toDate()).fromNow();
+    const { data: configuration } = useConfiguration(prebuild.configurationId);
```

Comment:
If I delete repository, it will try refetch with 404 with GetConfiguration (not sure what we can do for it if not change proto define)

Maybe we can do it like SCM service, to add configuration_name a field of prebuild?